docker/packager/binary/build.sh

```diff
@@ -35,6 +35,22 @@ ninja $NINJA_FLAGS clickhouse-bundle
 ccache --show-config ||:
 ccache --show-stats ||:
 
+# Also build fuzzers if any
+FUZZER_TARGETS=$(find ../src -name '*_fuzzer.cpp' -execdir basename {} .cpp ';' | tr '\n' ' ')
+
+mkdir -p /output/fuzzers
```

Comment:
Probably we need the same changes for `deb/build.sh`.

Reply:
Sure